tests/SConscript

```diff
@@ -1,4 +1,5 @@
 # -*- python -*-
 from lsst.sconsUtils import scripts
 scripts.BasicSConscript.pybind11(['testLib'])
-scripts.BasicSConscript.tests(noBuildList=['testLib.cc'])
```

Comment:
How did this work before?  It seems like it would have always failed if the library is not built.